src/workspaces/cookiefactoryv2/CookieFactoryDemo/src/config/sites.ts

```diff
@@ -1,37 +0,0 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 2023
-// SPDX-License-Identifier: Apache-2.0
-import type { SiteConfig } from '@/lib/types';
-
-export const WORKSPACE_ID = '__FILL_IN__';
-
-const sites: SiteConfig[] = [
-  {
-    awsConfig: {
-      sceneId: 'CookieFactory',
-      workspaceId: WORKSPACE_ID
-    },
-    id: crypto.randomUUID(),
-    location: '1 Main Street, Bakersville, NC, USA',
-    name: 'Bakersville Central'
-  },
-  // {
```

Comment:
oh, I don't remember whether I removed this locally and forgot to revert... I think having multiple sites is fine if you want to bring it back

Reply:
I just removed this file in favor of a `sites.template.ts`. made a not about it in the README update.